docs/source/gbd2019_models/causes/maternal_disorders/index.rst

```diff
@@ -202,12 +202,14 @@ Assumptions and Limitations
 
 - We assume that all maternal disorders deaths occur at birth. For deaths due to late maternal disorders, this will result in earlier maternal deaths in our model than in reality, which may overestimate the impact of maternal mortality on infant outcomes such as breastfeeding behaviors/orphanhood when we model maternal-child dyads. We may overestimate YLLs due to late maternal deaths and underestimate YLLs due to abortions, miscarriages, and ectopic pregnancies.
 
-- We assume that all maternal disorders morbidity *begins* at birth and persists for one year after birth. For morbidity due to hypertensive disorders and maternal sepsis may occur prior to when we model it. For sequelae that persist for longer than one year (ex: long term sequelae of eclampsia, infertility, fistula), we will underestimate YLDs.
+- We assume that all maternal disorders morbidity *begins* at birth and persists for one year after birth (but is accrued during a single week). For morbidity due to hypertensive disorders and maternal sepsis may occur prior to when we model it. For sequelae that persist for longer than one year (ex: long term sequelae of eclampsia, infertility, fistula), we will underestimate YLDs.
```

Comment:
```suggestion
- We assume that all maternal disorders morbidity *begins* at birth and persists for one year after birth (but is accrued during a single week). Morbidity due to hypertensive disorders and maternal sepsis may occur prior to when we model it. For sequelae that persist for longer than one year (ex: long term sequelae of eclampsia, infertility, fistula), we will underestimate YLDs.
```

Is it implied here that none of the sequelae persist for less than one year?

Reply:
No, we calculate maternal disorders YLDs according to the GBD estimates for YLDs rather than using disability weights by calculating the average amount of YLDs per incident case. However, since GBD estimates are specific to a given year, we therefore do not include disability due to longer lasting sequelae. We should appropriately capture disability due to more acute conditions, though. 

https://vivarium-research.readthedocs.io/en/latest/gbd2019_models/causes/maternal_disorders/index.html#years-lived-with-disability